src/sentry/api/endpoints/system_options.py

```diff
@@ -56,8 +61,20 @@ def get(self, request: Request) -> Response:
 
         return Response(results)
 
-    def put(self, request: Request):
+    def has_permission(self, request: Request):
         if not request.access.has_permission("options.admin"):
+            return False
+
+        for k, v in request.data.items():
+            if v and isinstance(v, str):
+                v = v.strip()
```

Comment:
Does this strip() do anything as `v` is never put back into the dictionary?

Reply:
It does not, I was just matching the other code and didn't think to remove it, fixed.